dffml/df/memory.py

```diff
@@ -1065,7 +1072,8 @@ class MemoryOrchestratorConfig(BaseOrchestratorConfig):
     """
 
 
-class MemoryOrchestratorContextConfig(NamedTuple):
+@config
+class MemoryOrchestratorContextConfig:
```

Comment:
We'll need to add

```python
@entrypoint("memory")
class MemoryOrchestrator(BaseOrchestrator, BaseMemoryDataFlowObject):
    CONFIG = MemoryOrchestratorConfig
    CONTEXT = MemoryOrchestratorContext
```

You'll also need to set the `CONFIG` property for all the non-`Context` classes. So anything that is a subclass of `BaseMemoryDataFlowObject`.

Reply:
Just to clarify is this required for this merge request?

Reply:
Ya, we'll need to remove the `args()` and `config()` methods as well, and this is what takes their place.

Reply:
Got it thanks @pdxjohnny